ui/app/components/ProfileComponent/ProfileComponent.js

```diff
@@ -144,7 +144,7 @@ class ProfileComponent extends React.Component {
               <div className={`${styles.itemContainer} ${(l.name === selectedTraceName && this.props.profile.start === selectedProfile) && styles.highlighted}`}
               key={l.name}>
                 <Link
-                  to={loc => ({ pathname: `/profiler/profile-data/${l.name}`, query: { ...loc.query, profileStart: this.props.profile.start, profileDuration: this.props.profile.duration }})}>
```

Comment:
have a route that ensures that /profiler/profile-data routes to /profile/profile-data. Add comments in routes wherever we are doing something for backward compatibility reasons

Reply:
Added the redirect route along with a comment